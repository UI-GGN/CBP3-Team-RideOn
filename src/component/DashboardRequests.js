import React from "react";
import BreadCrumb from "./BreadCrumb";

function DashboardRequests() {
  const obj = {name: 'Dashboard', type: 'Requests'};
  return <BreadCrumb data={obj} />;
}

export default DashboardRequests;
