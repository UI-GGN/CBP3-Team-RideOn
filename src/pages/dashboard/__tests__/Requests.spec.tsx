import {render} from "@testing-library/react";
import DashboardRequests from "../Requests";
import Breadcrumb from "../../../component/BreadCrumb";
import React from "react";
jest.mock(".../../../component/BreadCrumb");
describe("Requests test", () => {
  test("renders the component", async () => {
    render(<DashboardRequests />);

    expect(Breadcrumb).toBeCalledWith({data: {name: "Dashboard", type: "Requests"}}, {});
  });

  it("renders correctly", () => {
    const {baseElement} = render(<DashboardRequests />);

    expect(baseElement).toMatchSnapshot();
  });
});
