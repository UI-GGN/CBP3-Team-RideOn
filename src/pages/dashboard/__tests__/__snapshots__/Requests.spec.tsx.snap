// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Requests test renders correctly 1`] = `
<body>
  <div>
    <div
      class="breadcrumb"
    />
  </div>
</body>
`;
